data/abilities/collection/4e97e699-93d7-4040-b5a3-2e906a58199e.yml

```diff
@@ -19,9 +19,6 @@
     windows:
       psh:
         command: |
-          Copy-Item #{host.file.sensitive} #{host.dir.staged}
-      cmd:
-        command: |
-          copy #{host.file.sensitive} #{host.dir.staged}
```

Comment:
cmd should be able to handle spaces

Reply:
Please test this, I didn't remove it based on conjecture, but rather encountering an issue with files w/ spaces and wasting lots of time trying to implement a workaround.  Also note, it is not an issue with cmd specifically but rather how Caldera uses it (and if I recall correctly, how Caldera escapes quotes) that makes handling spaces difficult.  You would think simply adding quotation marks around #{host.file.sensitive} would resolve it, but I assure you and much to my frustration, it does not.  Finally concluded that PowerShell had a more sane approach to spacing and went w/ that.  ;)